### ISSUE TITLE
[deeplab v3p] Learning progress is too slow

### ISSUE THREAD

**Issue by hshintaro575** [tensorflow/models#4386]
I'm conducting learning of deeplab from zero base by deleting init_fn in train.py.
Although learning is started,  learning progress is too slow.

I adopted default parameter. The parameter is below:

`--model_variant=xception_65   --atrous_rates=6   --atrous_rates=12   --atrous_rates=18   --output_stride=16   --decoder_output_stride=4 ...`

The loss value provided by Deeplab is below :

10 step:  loss = 5.8
100 step:  loss = 5.2
1000 step:  loss = 5.0
2000 step:  loss = 5.0
3000 step:  loss = 4.9
4000 step:  loss = 4.8
5000 step:  loss = 4.8
6000 step:  loss = 4.7
...

Is this progress correct?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by zhoufengbuaa** [comment]
@hshintaro575 I couldn't reproduce the performance shown in this article, and I couldn't train the results using the learning rate and other hyper parameters given in the code.

**Comment by hshintaro575** [comment]
I'm sorry for the late replay.

I have confirmed that local_test.sh runed.
After that, I tried learning from zero-base. I do not use original data.

Changed the following source code of train.py

'''
slim.learning.train(
    train_tensor,
    logdir=FLAGS.train_logdir,
    log_every_n_steps=FLAGS.log_steps,
    master=FLAGS.master,
    number_of_steps=FLAGS.training_number_of_steps,
    is_chief=(FLAGS.task == 0),
    session_config=session_config,
    startup_delay_steps=startup_delay_steps,
    #init_fn=train_utils.get_model_init_fn(
    #    FLAGS.train_logdir,
    #    FLAGS.tf_initial_checkpoint,
    #    FLAGS.initialize_last_layer,
    #    last_layers,
    #    ignore_missing_vars=True),
    summary_op=summary_op,
    save_summaries_secs=FLAGS.save_summaries_secs,
    save_interval_secs=FLAGS.save_interval_secs)
'''
By eliminating init_fn, learning can be done without a checkpoint file.
However, it does not converge.

I adopted default parameter. The parameter is below:

'--model_variant=xception_65 --atrous_rates=6 --atrous_rates=12 --atrous_rates=18 --output_stride=16 --decoder_output_stride=4 ...'

The following is the progress of learning.
'''
INFO:tensorflow:global step 14770: loss = 6.5108 (0.816 sec/step)
INFO:tensorflow:global step 14780: loss = 6.1914 (0.818 sec/step)
INFO:tensorflow:global step 14790: loss = 6.7189 (0.810 sec/step)
INFO:tensorflow:global step 14800: loss = 6.1251 (0.814 sec/step)
INFO:tensorflow:global step 14810: loss = 6.6132 (0.808 sec/step)
INFO:tensorflow:global step 14820: loss = 6.3956 (0.799 sec/step)
INFO:tensorflow:global step 14830: loss = 5.7623 (0.821 sec/step)
INFO:tensorflow:global step 14840: loss = 5.9424 (0.833 sec/step)
INFO:tensorflow:global step 14850: loss = 5.7299 (0.805 sec/step)
INFO:tensorflow:global step 14860: loss = 5.9387 (0.801 sec/step)
'''

Is this progress correct?

**Comment by tt-leader** [comment]
I trained the model from scratch too, using the default parameters and batch_size=1.
After 50k steps, the loss turns to 3.8 around.

**Comment by hshintaro575** [comment]
Thank you for your report.
I am batch size 4.
I think that this model can not use well inference if this model is loss upper than 1.0.
I will try learning ....

**Comment by MarkYangjiayi** [comment]
@hshintaro575 Hello, I am also experiencing the same problem as you did. Wondering if you ever fixed this problem? Does the loss go down after further training or did you change some parameters? Thanks in advance

**Comment by vickylibra** [comment]
@hshintaro575 Hi, I also have the same problem as you did. Do you find any solution?  Or anyone knows the reason? Please help me, thank you all!
